### PR TITLE
Improve support for the SUTON STB3L-125-ZJ DIN rail RCBO (TZE204_wbha…

### PR DESCRIPTION
**Pull request by octa22**
This PR improves support for the SUTON STB3L-125-ZJ DIN rail RCBO (TZE204_wbhaespm)

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -5529,7 +5669,7 @@ const definitions: DefinitionWithExtend[] = [
         vendor: 'SUTON',
         description: 'Zigbee DIN RCBO energy meter',
         fromZigbee: [tuya.fz.datapoints],
-        toZigbee: [tuya.tz.datapoints],
+        toZigbee: [tzLocal.TZE204_threshold, tuya.tz.datapoints],
```
Comment (Koenkk) [inline src/devices/tuya.ts:5672]: Please use just `tuya.tz.datapoints` instead of `TZE204_threshold`

Comment (octa22) [reply src/devices/tuya.ts:5672]: OK, you mean to implement the logic in tuya.valueConvertor? Is there an example how to implement a setting logic for several exposed attributes behind a single dataPoint? Thanks

Comment (Koenkk) [reply src/devices/tuya.ts:5672]: Yes, here is an example to send data points directly from the converter (https://github.com/Koenkk/zigbee-herdsman-converters/blob/615eb00f66267429fa8434a233b39f2259af3d1a/src/lib/tuya.ts#L977). With this you ca also send multiple.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**